### PR TITLE
raphael: Few power related changes

### PR DESCRIPTION
**Pull request by penglezos**
These commits results in properly loading mi_thermald without any issues while fixing high active drain and abnormal phone temperature on light tasks.

### CODE REVIEW

File: rootdir/etc/init.target.rc [hunk 1]
```diff
@@ -27,6 +27,8 @@
 #
 #
 
+import /vendor/etc/init/init.mi_thermald.rc
```
Comment (justks) [inline rootdir/etc/init.target.rc]: Can remove this
Should start on its own, no need to import.

File: rootdir/etc/init.target.rc [hunk 2]
```diff
@@ -75,6 +77,8 @@ on post-fs-data
     mkdir /tombstones/dsps 0771 system system
     mkdir /data/vendor/hbtp 0750 system system
     mkdir /data/vendor/nnhal 0700 system system
+    mkdir /data/vendor/thermal 0771 root system
```
Comment (justks) [inline rootdir/etc/init.target.rc]: Set these in init.xiaomi.rc instead

Comment (penglezos) [reply rootdir/etc/init.target.rc]: Done, thanks for pointing this out.

File: rootdir/etc/init.target.rc (hunk 1 above)
Comment (penglezos) [reply rootdir/etc/init.target.rc]: I'm not so sure to be honest so I kept it there. Should be fine I guess.

Comment (justks) [reply rootdir/etc/init.target.rc]: https://github.com/ArrowOS-Devices/android_device_xiaomi_vayu/commit/6311b55ea1498aeafd0aa26275c2e24752eab99e

Comment (justks) [reply rootdir/etc/init.target.rc]: It starts on its own, so putting an import here will cause it to spit out as a duplicate definition

Comment (penglezos) [reply rootdir/etc/init.target.rc]: Done! Checked my logs too and this `init: /vendor/etc/init/init.mi_thermald.rc: 8: ignored duplicate definition of service 'mi_thermald'` appears! Did you checked if abnormal high temperature is fixed for you?

Comment (justks) [reply rootdir/etc/init.target.rc]: I don't usually face abnormal high temperature in my usage

Comment (PptO07) [reply rootdir/etc/init.target.rc]: > I don't usually face abnormal high temperature in my usage

I am facing a bit high temps on normal usage too hope this commit fixes it

Comment (penglezos) [reply rootdir/etc/init.target.rc]: > > I don't usually face abnormal high temperature in my usage
> 
> I am facing a bit high temps on normal usage too hope this commit fixes it

If you are building yourself you can include the pathset and report back.

Comment (PptO07) [reply rootdir/etc/init.target.rc]: > > > I don't usually face abnormal high temperature in my usage
> > 
> > 
> > I am facing a bit high temps on normal usage too hope this commit fixes it
> 
> If you are building yourself you can include the pathset and report back.

Is the minimal cpu set commit done git reset i was unable to find it on arrow device org tree

**Comment by KujouYuko** [comment]
Done.

**Pull request closed without merging**